lib/jackal-kitchen/tester.rb

```diff
@@ -200,6 +202,30 @@ def insert_kitchen_ssh(path)
         end
       end
 
+      def update_spec_helpers(path)
+        spec_helpers = []
+        Find.find(path) do |location|
+          spec_helpers << location if location =~ /spec_helper.rb/
+        end
+        spec_helpers.each do |spec_helper|
+          content = File.readlines(spec_helper)
+          open(spec_helper, 'a') { |f|
```

Comment:
lame style comment... usually with multi-line blocks, I believe it's a bit more conventional to use do/end.
Also, the heredoc will make it a bit easier to read / change IMHO.

Oh, also, you can just use Dir.glob('**/spec_helper.rb'), instead of find (don't have to require a non-base module)
also will it be possible for the spec_helpers to already contain an `RSpec.configure` block?  If so, will this cause issues?

The style stuff is obviously not critical, so I'll go ahead and merge if @cwjohnston doesn't have anything to add...
do I need anything special to run the tests?  just running jackal-test should suffice?